aws/table_aws_api_gatewayv2_integration.go

```diff
@@ -0,0 +1,274 @@
+				Description: "Specifies the integration's HTTP method type.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "integration_type",
+				Description: "Represents an API method integration type.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "integration_uri",
+				Description: "A string representation of a URI with a length between [1-2048]. For a Lambda integration, specify the URI of a Lambda function. For an HTTP integration, specify a fully-qualified URL. For an HTTP API private integration, specify the ARN of an Application Load Balancer listener, Network Load Balancer listener, or AWS Cloud Map service. If you specify the ARN of an AWS Cloud Map service, API Gateway uses DiscoverInstances to identify resources. You can use query parameters to target specific resources. To learn more, see DiscoverInstances (https://docs.aws.amazon.com/cloud-map/latest/api/API_DiscoverInstances.html). For private integrations, all resources must be owned by the same AWS account.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "connection_id",
+				Description: "The ID of the VPC link for a private integration. Supported only for HTTP APIs.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "connection_type",
+				Description: "Represents a connection type.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "timeout_in_millis",
+				Description: "Custom timeout between 50 and 29,000 milliseconds for WebSocket APIs and between 50 and 30,000 milliseconds for HTTP APIs. The default timeout is 29 seconds for WebSocket APIs and 30 seconds for HTTP APIs.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "integration_subtype",
+				Description: "A string with a length between [1-128].",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "api_gateway_managed",
+				Description: "Specifies whether an integration is managed by API Gateway. If you created an API using using quick create, the resulting integration is managed by API Gateway. You can update a managed integration, but you can't delete it.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "content_handling_strategy",
+				Description: "Specifies how to handle response payload content type conversions. Supported only for WebSocket APIs.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "credentials_arn",
+				Description: "Represents an Amazon Resource Name (ARN).",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "integration_response_selection_expression",
+				Description: "An expression used to extract information at runtime. See Selection Expressions(https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-websocket-api-selection-expressions.html#apigateway-websocket-api-apikey-selection-expressions for more information.",
```

Comment:
String columns should be in alphabetical order and json column should in the last.